src/client/package.json

```diff
@@ -53,7 +53,7 @@
   "dependencies": {
     "@adaptive/hydra-platform": "^3.120.0",
     "@finos/fdc3": "^1.2.0",
-    "@openfin/core": "^27.71.21",
+    "@openfin/core": "^29.0.0",
```

Comment:
should match openfin runtime ["latest" in npm is 29](https://www.npmjs.com/package/@openfin/core/v/29.73.14) but later versions are available .. we are limiting to v29 for now, anyway